### DISCUSSION TITLE
A Hover Effect must be added to the Person Icon to Update Photo

### DISCUSSION THREAD

**Discussion by warmachine028** [warmachine028/uemcrp#11] [Ideas]
- When the user hovers over user Icon, An overlay must be visible which would have a camera icon.

**Comment by Sohelahmed-molla** [comment]
For camera icon we can use  materialize css . 
![Screenshot (132)](https://user-images.githubusercontent.com/73135042/187026346-253afbd6-1cc0-4081-887c-ed3d9a8f495d.png)

**Reply by warmachine028** [reply]
Great Idea. But as we are already working with material UI so. We can find the camera icon from here 

https://mui.com/material-ui/material-icons/#main-content
